examples/all-clusters-app/linux/main-common.cpp

```diff
@@ -208,9 +208,9 @@ void ApplicationInit()
     Clusters::UnitLocalization::TempUnitEnum supportedUnits[2] = { Clusters::UnitLocalization::TempUnitEnum::kFahrenheit,
                                                                    Clusters::UnitLocalization::TempUnitEnum::kCelsius };
     DataModel::List<Clusters::UnitLocalization::TempUnitEnum> unitsList(supportedUnits);
-    VerifyOrDie(Clusters::UnitLocalization::UnitLocalizationServer::Instance().SetSupportedTemperatureUnits(unitsList) ==
+    VerifyOrDie(Clusters::UnitLocalization::UnitLocalizationCluster::Instance().SetSupportedTemperatureUnits(unitsList) ==
```

Comment:
Do not rename the class. For backwards compatibility we expect:

- we will have a UnitLocalizationCluster
- we will also have a `UnitLocalizationServer` in codegenintegration (and the -server include) that will be a backwards compatibility API shim

The intent is to make migration to new versions as easy as possible for applications. We expect to have the old header,  that includes `CodgenIntegration.h` and that codegenintegration.h defines the `Server`